docker/entrypoint.sh

```diff
@@ -17,6 +17,14 @@ rm -rf _tmp_ssh _tmp_https
 echo "Copying serverfiles overrides..."
 rsync --verbose --archive $home/serverfiles_override/ $gmodroot/
 
+# Any additional files
+cp $home/_gluatest_artifacts/_gluatest_artifacts/* $gmodroot/
+
+for file in $gmodroot/*.tar.gz; do \
+    tar -xzf "$file" -C "$gmodroot" && \
+    rm -f "$file"; \
```

Comment:
I've found that having verbose output on commands like this helps a lot with debugging. I also prefer long-form parameters so it's easier to parse through 🙌 

```suggestion
    tar --extract --verbose --ungzip --file="$file" --directory="$gmodroot" && \
    rm --force --verbose "$file"; \
```